README.md

```diff
@@ -238,7 +266,7 @@ helm install crud snowdrop/spring-boot-example-crud
 
 ### spring-boot-example-configmap
 
-This chart deploys the example from [the repository](https://github.com/snowdrop/configmap-example). This example uses a ConfigMap to externalize configuration. ConfigMap is an object used by OpenShift to inject configuration data as simple key and value pairs into one or more Linux containers while keeping the containers independent of OpenShift.
+This chart deploys the example from [the repository](https://github.com/snowdrop/configmap-example). This example uses a ConfigMap to externalize configuration. ConfigMap is an object used by OpenShift and Kubernetes to inject configuration data as simple key and value pairs into one or more Linux containers while keeping the containers independent.
```

Comment:
"... to externalize the configuration". Idem for "The ConfigMap...."

and

"while keeping the containers independent." From what ?

Reply:
I deleted this sentence as I don't think it adds value here.